aws-android-sdk-core/src/main/java/com/amazonaws/auth/CognitoCredentialsProvider.java

```diff
@@ -449,6 +449,12 @@ public Date getSessionCredentitalsExpiration() {
         }
     }
 
+    @Deprecated
+    /** Use {@link #getSessionCredentialsExpiration()} instead. */
```

Comment:
generally the annotation (`@Deprecated`) comes right before the method signature.

```suggestion
    /** 
     * @deprecated Use {@link #getSessionCredentialsExpiration()} instead. 
     */
    @Deprecated
```